modules/installation-and-upgrade/pages/hardware-requirements.adoc

```diff
@@ -165,39 +165,41 @@ For more information about managing a large number of clients, see xref:speciali
 | Minimum 40{nbsp}GB
 
 |
-| [path]``/srv``
+| [path]``/var/lib/containers/storage/volumes/srv-www``
```

Comment:
Do you really want to add this one here? `/srv` could contain some data in the past, but not the `srv-www` volume contains the bootstrap repos, not much more.

Reply:
Left as is after our discussion. Boostrap content, distributions, and containers are stored here so it should be considered.